app/teachingCall/css/teaching-call-form.css

```diff
@@ -886,7 +886,8 @@ li.active-term .dropdown-term-row {
 }
 
 .search-course-container .dropdown-menu {
-	width: 26%;
+	min-width: 26%;
+	width: auto!important;
```

Comment:
Usage of the !important flag should only be used as a last resort, it causes complications in CSS - is it possible to rewrite this without it?